mappings/86dc46f55f038356878e84fa98169c75ebd86db1a5894eb68d1cd91750584c.json

```diff
@@ -0,0 +1,64 @@
+{
+    "subject": "86dc46f55f038356878e84fa98169c75ebd86db1a5894eb68d1cd91750584c",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://www.itsapixel.com",
```

Comment:
Hi @kylebehse,

please put policyId to the website or remove the field.

Thanks!

Reply:
Thanks @VladislavKudrin ,

I'm looking into what the requirements are for the policyId. The token will be used to support a larger company so I do not want any mention of it directly on the homepage but instead on a page like:
https://www.itsapixel.com/token-policy
I can update the url -> value to reflect that but was wondering if you had a template or any references to what is required to be on that page in order to meet this reqirement?

Thanks again,
Kyle

Reply:
@VladislavKudrin ,

I couldn't find a requirements template so I just asked GPT. It gave me a bunch of suggestions, so I've started a document. Can you let me know if I'm on the right track?

[https://www.itsapixel.com/its-a-pixel-token-policy](url)

Note: It's just going to be used as a loyalty program. Like AirMiles that can be swapped on the market if someone doesn't want them. 

I'm just curious, if I continue with this template, will that meet the requirements?

Thanks so much for your time!

Reply:
@kylebehse,

the page should be accessible from the submitted URL. For example on https://www.itsapixel.com there is a link to token-policy page

Reply:
@VladislavKudrin , ok great, thanks, I'll restructure and get back to you. Cheers!